docs/snapshotting/snapshot-support.md

```diff
@@ -128,6 +128,12 @@ The snapshot functionality is still in developer preview due to the following:
 
 ### Limitations
 
+- Currently on aarch64 platforms only lower 128 bits of any register are
+  saved due to the limitations of `get/set_one_reg` from `kvm-ioctls` crate
+  that Firecracker uses to interact with KVM. This creates an issue with
+  newer aarch64 CPUs with support for registers with width greater than
+  128 bits, because these registers will be truncated before being stored
+  in the snapshot. This can lead to uVM failure if restored from such snapshot.
```

Comment:
Can this risk be mitigated? How can a user ensure that no wide registers are being used before taking the snapshot? For example, don't do any SVE math operations prior to taking the snapshot?

Reply:
I don't think this is possible to know exactly if a program uses SVE or not. Some programs may decide it at run time.

Reply:
Added a couple sentences about possible mitigations.